### PR TITLE
[UWP] Renames from Background Image spec updates

### PR DESCRIPTION
**Pull request by RebeccaAnne**
This includes the renames, but not the functionality changes, for #2801.

Verified test results and updated changed files.

### CODE REVIEW

**Comment by nesalang** [comment]
cortana-skills.Tests.BackgroundImage.VerticalAndHorizontalAlignment.png shows that a background image stretches its container

**Comment by RebeccaAnne** [comment]
Yeah, you're right. It's a known issue tracked by [https://github.com/microsoft/AdaptiveCards/issues/2697](https://github.com/microsoft/AdaptiveCards/issues/2697).

---
In reply to: [493519697](https://github.com/microsoft/AdaptiveCards/pull/2928#issuecomment-493519697) [](ancestors = 493519697)

File: source/uwp/Renderer/lib/TileControl.cpp [hunk 1]
```diff
@@ -196,9 +196,9 @@ namespace AdaptiveNamespace
         // the container until we get the right dimensions.
         if (m_imageSize.Width != 0 && m_imageSize.Height != 0)
         {
-            switch (mode)
```
Comment (nesalang) [inline source/uwp/Renderer/lib/TileControl.cpp]: no case for "Cover"?

Comment (RebeccaAnne) [reply source/uwp/Renderer/lib/TileControl.cpp]: So, this is just a rename, and I'm not 100% familiar with the code here, but, i believe this is the code that handles tiling the images so cover (previously known as stretch) isn't a relevant case.  

---
In reply to: [285204074](https://github.com/microsoft/AdaptiveCards/pull/2928#discussion_r285204074) [](ancestors = 285204074)

**Pull request merged**